hugegraph-server/hugegraph-dist/docker/example/docker-compose-cassandra.yml

```diff
@@ -39,7 +39,7 @@ services:
       retries: 3
 
   cassandra:
-    image: cassandra:4
+    image: cassandra:3.11
```

Comment:
why we setback to the version 3.x?

Reply:
https://github.com/apache/incubator-hugegraph/pull/2374
I remember we rollback cassandra to 3.11 and it is approved by one reviewer.
But i find the pr it suspend just now.
I will adjust the yaml with license together.

Reply:
> #2374 I remember we rollback cassandra to 3.11 and it is approved by one reviewer. But i find the pr it suspend just now. I will adjust the yaml with license together.

the problem has been solved (temporarily?), so no need to rollback it for now~